requirements/libraries.txt

```diff
@@ -24,7 +24,12 @@ govdelivery==1.4.0
 Jinja2==3.1.3
 lxml==5.1.0
 opensearch-py==2.4.2
-psycopg2-binary==2.9.9
+# psycopg2 > 2.9.6 causes cf.gov to hang locally for people with on-network
+# Macs, so we'll pin to 2.9.6 for now.
+# See https://github.com/psycopg/psycopg2/issues/1645
+# and https://github.com/dbt-labs/dbt-core/issues/9303#issuecomment-1864934933
+# for the details.
```

Comment:
```suggestion
```

I don't think it's necessary to document this in the requirements file since we'll have the git blame / PR reference.

Reply:
Comment removed in 0652fc5